### ISSUE TITLE
Feature subscriptions aren't accurate (or are broken)

### ISSUE THREAD

**Issue by farski** [integrations/slack#484]
If you run these two commands

- `/github subscribe acme/server issues`
- `/github subscribe acme/server commits`

and then run `/github unsubscribe acme/server issues` to unsubscribe for a single feature, the app responds with a message like: 

[#some-channel](example.com) will get notifications from [acme/server](example.com) for: `pulls`, `deployments`, `statuses`, `public`, `commits`

Which seems to indicate that the subscription includes a number of features that were never subscribed to. At this point the only subscribed feature should be `commits`.

**Comment by bkeepers** [comment]
Hey @farski, thanks for the issue.  Currently the "default" settings are enabled whenever a subscription is first created, even if you specify other settings. I agree this is counterintuitive. It was intentionally, but I think it's fair to call it a `bug` from the user's perspective.

**Comment by farski** [comment]
@bkeepers Got it; that might mean the other issue I filed #483 is just getting bit by the same issue. Is the correct thing to do (for the time being) to subscribe, and then _unsubscribe_ from the default settings that I don't want?

eg
`/github subscribe acme/server`
`/github un subscribe acme/server pulls, deployments, statuses, public, issues`

**Comment by farski** [comment]
I believe that it is

**Comment by lwasylkowski** [comment]
I believe that it is

**Comment by farski** [comment]
It is

**Comment by stephen-pugliese** [comment]
Is this ever going to be fixed? It’s been open over a year and is a feautre that is widely available.

**Comment by farski** [comment]
Yes, the [default values](https://github.com/integrations/slack/blob/master/lib/models/subscription.js#L21) on the model are still very counterintuitive. This is a non-breaking change, as it only affects new subscriptions (though it is a behavior change).

**Comment by sohussain** [comment]
`unsubscribe` does not seem to be working. Tried with a private repo.
Unsubscribed for `pulls` but we're still getting notifications for pull requests. 😕

**Comment by farski** [comment]
If it hasn't been fixed and closed I have to assume it's still relevant.

**Comment by farski** [comment]
Running `/github subscribe acme/server issues` now responds with a message like ":white_check_mark: Subscribed to https://github.com/acme/server. This channel will receive notifications for issues, pulls, commits, releases, deployments"

It's still not intuitive that trying to subscribe to a specific feature set doesn't work, but it's at least a lot more obvious now what's happening. I would still prefer that if specific features are explicitly listed when subscribing, only those feature are subscribed to. The aggressive defaults don't seem to provide much valuable in this case.
